core/src/main/PoliticalMap.kt

```diff
@@ -17,9 +19,9 @@ class PoliticalMap private constructor(
 
     class Builder {
 
-        private val continents = ContinentSet()
+        private val continents = HashSet<Continent>()
```

Comment:
Esto podría ser
```suggestion
        private val continents = mutableSetOf<Continent>()
```
así no nos preocupamos por que sea un `HashSet` u otro tipo de `MutableSet`.